assets/js/components/date-range-selector.js

```diff
@@ -99,12 +109,13 @@ class DateRangeSelector extends Component {
 		return (
 			<Select
 				enhanced
+				se
 				className="mdc-select--minimal"
 				name="time_period"
 				label=""
 				onEnhancedChange={ this.handleSelection }
 				options={ options }
-				value={ applyFilters( this.dateRangeHook, __( 'Last 28 days', 'google-site-kit' ) ) }
```

Comment:
@tofumatt Can you clarify why you're changing this here? This seems part of #447 more than this issue. If there were specific problems that this caused when rearranging the assets, would be great if you could clarify this.

Either way, I agree with @aaemnnosttv that we should either keep this around for now or eliminate the filter everywhere.

Reply:
Sure thing. I changed it to tidy up slightly but I don't think it's required here. I'll change it back 👍

Reply:
(Side note: `getCurrentDateRange()` just calls `applyFilters( this.dateRangeHook, __( 'Last 28 days', 'google-site-kit' ) )`, so I just changed it here to tidy up the code, but still: I'll change it back to keep the PR a bit more atomic.

Reply:
Oh, and this is just using `state` to force a component update when the date is updated, which otherwise wouldn't happen here. The values are all still the same.